src/ResourceManager/Sql/Commands.Sql/Database Backup/Services/AzureSqlDatabaseBackupAdapter.cs

```diff
@@ -379,23 +493,34 @@ internal AzureSqlDeletedDatabaseBackupModel GetDeletedDatabaseBackup(string reso
         /// <returns>Restored database object</returns>
         internal AzureSqlDatabaseModel RestoreDatabase(string resourceGroup, DateTime restorePointInTime, string resourceId, AzureSqlDatabaseModel model)
         {
-            DatabaseCreateOrUpdateParameters parameters = new DatabaseCreateOrUpdateParameters()
+            if (model.CreateMode.Equals("RestoreLongTermRetentionBackup") && CultureInfo.CurrentCulture.CompareInfo.IndexOf(resourceId, "/providers/Microsoft.Sql", CompareOptions.IgnoreCase) >= 0)
```

Comment:
`OrdinalIgnoreCase`  is the right comparison to use

Reply:
Fixed. Will be in next commit.